Rego/EXOConfig.rego

```diff
@@ -477,8 +487,9 @@ tests[{
     "ReportDetails" : ReportDetailsString(Status, ErrorMessage),
     "RequirementMet" : Status
 }]{
-    ErrorMessage := "Allow-list is in use"
-    Status := count(ConnFiltersWithIPAllowList) == 0
+    ConnFilterPolicies := ConnFiltersWithIPAllowList
```

Comment:
Shouldn't `ConnFilterPolicies` be the ActualValue in line 286 instead of `input.conn_filter`?